### ISSUE TITLE
Multi user login in with the same user will  force exit

### ISSUE THREAD

**Issue by zxbfeng** [apache/pulsar-manager#275]
Multi users login in with the same account will  force exit.

**Comment by tuteng** [comment]
Yes, this is the expected behavior. One account and password cannot be logged in at multiple places at the same time. If you want to use multiple accounts, you should use super users to create different accounts. @zxbfeng https://localhost:9527/#/users

**Comment by zxbfeng** [comment]
Thank you ! https://localhost:9527/#/users  this  page  always flash by，try many times can remain at this page, and I add a new user account  then login in , the page is http://localhost:9527/#/404

**Comment by sde-melo** [comment]
Same behaviour for me. Besides a tenant with the same name is created.

**Comment by tuteng** [comment]
@zxbfeng @sde-melo Which version is used, is it the latest code or 0.1.0? The latest version already supports user management.

**Comment by zxbfeng** [comment]
the latest. I use the user management to add some other user accounts, but the new added user when I use it to login in ,it displays the&nbsp;http://hostname/#/404 pages




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "guangning"<notifications@github.com&gt;; 
发送时间: 2020年3月20日(星期五) 下午3:47
收件人: "apache/pulsar-manager"<pulsar-manager@noreply.github.com&gt;; 
抄送: "雨的记忆"<261045585@qq.com&gt;; "Mention"<mention@noreply.github.com&gt;; 
主题: Re: [apache/pulsar-manager] Multi user login in with the same user will  force exit (#275)





 
@zxbfeng @sde-melo Which version is used, is it the latest code or 0.1.0? The latest version already supports user management.
